iris-logo/README.md

```diff
@@ -0,0 +1,41 @@
+# The Iris Logo
+
+## Logo
+
+The official logo can be obtained, in SVG form, via
+
+    latexmk logo
+
+This will produce the SVG by first compiling to DVI then to SVG using `dvisvgm`.
+Alternatively, by running
+
+    latexmk -pdf logo
+
+one gets a PDF and an SVG obtained by running `pdf2svg`.
```

Comment:
When should I use this alternative? Will this produce a different result? Which method is used for the official files? It's a bit confusing to have two ways when it's not clear why one would use one or the other.

Reply:
My preference would be to go `latexmk logo` to produce the SVG, and `pdflatex logo` to produce a PDF (if needed)

Reply:
If you prefer to call convert as well all in one command, one can ditch latexmk and prepare an equivalent make file.

Reply:
I don't have a strong preference either way here, I was just trying to understand the setup.^^ I find the documentation didn't answer the questions that came up when reading it, hence my comments.